packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/actions/shuffleThis.yml

```diff
@@ -12,16 +12,5 @@ initialState:
     - anchor: {line: 0, character: 0}
       active: {line: 0, character: 5}
   marks: {}
-finalState:
-  documentContents: hello
-  selections:
-    - anchor: {line: 0, character: 0}
-      active: {line: 0, character: 5}
-  thatMark:
-    - type: UntypedTarget
-      contentRange:
-        start: {line: 0, character: 0}
-        end: {line: 0, character: 5}
-      isReversed: false
-      hasExplicitRange: true
 fullTargets: [{type: primitive, mark: {type: cursor}, selectionType: token, position: contents, insideOutsideType: inside, modifier: {type: identity}, isImplicit: false}]
+thrownError: {name: Error}
```

Comment:
I was leaning towards a warning for reasons mentioned in other comment. Do you prefer error?

Reply:
updated